packages/codemods/src/codeMods/tests/mock/v9/mMakeStylesDeep.ts

```diff
@@ -0,0 +1,45 @@
+// @ts-ignore
```

Comment:
every single line is ignored.... can you use `// @ts-nocheck` ?

Reply:
Yup 👍